scripts/clean_images.py

```
@@ -0,0 +1,13 @@
+from constants import *
+import subprocess
+
+def clearContainers():
+    cmd = ["docker", "rmi", DLV_IMAGE_NAME]
```

Ruff (0.8.2)

`DLV_IMAGE_NAME` may be undefined, or defined from star imports

(F405)